src/genesisStates/ropsten.json

```diff
@@ -1,259 +1,259 @@
+  "0x00000000000000000000000000000000000000cc": "0x0",
+  "0x00000000000000000000000000000000000000cd": "0x0",
+  "0x00000000000000000000000000000000000000ce": "0x0",
+  "0x00000000000000000000000000000000000000cf": "0x0",
+  "0x00000000000000000000000000000000000000d0": "0x0",
+  "0x00000000000000000000000000000000000000d1": "0x0",
+  "0x00000000000000000000000000000000000000d2": "0x0",
+  "0x00000000000000000000000000000000000000d3": "0x0",
+  "0x00000000000000000000000000000000000000d4": "0x0",
+  "0x00000000000000000000000000000000000000d5": "0x0",
+  "0x00000000000000000000000000000000000000d6": "0x0",
+  "0x00000000000000000000000000000000000000d7": "0x0",
+  "0x00000000000000000000000000000000000000d8": "0x0",
+  "0x00000000000000000000000000000000000000d9": "0x0",
+  "0x00000000000000000000000000000000000000da": "0x0",
+  "0x00000000000000000000000000000000000000db": "0x0",
+  "0x00000000000000000000000000000000000000dc": "0x0",
+  "0x00000000000000000000000000000000000000dd": "0x0",
+  "0x00000000000000000000000000000000000000de": "0x0",
+  "0x00000000000000000000000000000000000000df": "0x0",
+  "0x00000000000000000000000000000000000000e0": "0x0",
+  "0x00000000000000000000000000000000000000e1": "0x0",
+  "0x00000000000000000000000000000000000000e2": "0x0",
+  "0x00000000000000000000000000000000000000e3": "0x0",
+  "0x00000000000000000000000000000000000000e4": "0x0",
+  "0x00000000000000000000000000000000000000e5": "0x0",
+  "0x00000000000000000000000000000000000000e6": "0x0",
+  "0x00000000000000000000000000000000000000e7": "0x0",
+  "0x00000000000000000000000000000000000000e8": "0x0",
+  "0x00000000000000000000000000000000000000e9": "0x0",
+  "0x00000000000000000000000000000000000000ea": "0x0",
+  "0x00000000000000000000000000000000000000eb": "0x0",
+  "0x00000000000000000000000000000000000000ec": "0x0",
+  "0x00000000000000000000000000000000000000ed": "0x0",
+  "0x00000000000000000000000000000000000000ee": "0x0",
+  "0x00000000000000000000000000000000000000ef": "0x0",
+  "0x00000000000000000000000000000000000000f0": "0x0",
+  "0x00000000000000000000000000000000000000f1": "0x0",
+  "0x00000000000000000000000000000000000000f2": "0x0",
+  "0x00000000000000000000000000000000000000f3": "0x0",
+  "0x00000000000000000000000000000000000000f4": "0x0",
+  "0x00000000000000000000000000000000000000f5": "0x0",
+  "0x00000000000000000000000000000000000000f6": "0x0",
+  "0x00000000000000000000000000000000000000f7": "0x0",
+  "0x00000000000000000000000000000000000000f8": "0x0",
+  "0x00000000000000000000000000000000000000f9": "0x0",
+  "0x00000000000000000000000000000000000000fa": "0x0",
+  "0x00000000000000000000000000000000000000fb": "0x0",
+  "0x00000000000000000000000000000000000000fc": "0x0",
+  "0x00000000000000000000000000000000000000fd": "0x0",
+  "0x00000000000000000000000000000000000000fe": "0x0",
+  "0x00000000000000000000000000000000000000ff": "0x0",
+  "0x874b54a8bd152966d63f706bae1ffeb0411921e5": "0xc9f2c9cd04674edea40000000"
 }
```

Comment:
Ok, number of "1" -> "0x1" balances stayed the same, single more funded account moved to bottom.